src/main.rs

```diff
@@ -532,10 +532,11 @@ fn run() -> Result<(), failure::Error> {
         let manifest = settings::toml::Manifest::new(config_path)?;
         let env = matches.value_of("env");
         let mut target = manifest.get_target(env)?;
+        let deploy_target = manifest.deploy_target(env)?;
 
         let verbose = matches.is_present("verbose");
 
-        commands::publish(&user, &mut target, verbose)?;
+        commands::publish(&user, &mut target, deploy_target, verbose)?;
```

Comment:
should we remove the `&mut target` here and also the `let mut target` creation above?

Reply:
no, those are still used for upload and build steps. i'd like to get to where we have logic for each of those configs, as well, though.